scheduler/db/scheduler.changelog-master.yaml

```diff
@@ -134,6 +139,11 @@ databaseChangeLog:
                   type: integer
                   constraints:
                     nullable: true
+              - column:
+                  name: guid
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
```

Comment:
I feel like we should get in the habit of using change sets even if we haven't officially released yet -- that way we can shake out any issues while we still have an escape hatch if we need it